layout.js

```diff
@@ -0,0 +1,18 @@
+function layout(title, content) {
+    return /*html*/ `<!DOCTYPE html>
+  <html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="an app for sharing quotations">
+    <title>${"Upload File"}</title>
```

Comment:
This is a placeholder, so should just be "title", which can then be an argument used for generating new pages (e.g. layout('signup', '<p>sign up</p>'))